alpine/alpine_test.go

```diff
@@ -1,43 +1,42 @@
-package alpine
+package alpine_test
 
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"io/ioutil"
+	"os"
 	"path"
+	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
 
 	"golang.org/x/xerrors"
 
+	"github.com/aquasecurity/vuln-list-update/alpine"
+
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 )
 
 type MockGitConfig struct {
-	cloneorpull  func(string, string) (map[string]struct{}, error)
-	remotebranch func(string) ([]string, error)
-	checkout     func(string, string) error
+	mock.Mock
 }
 
 func (mgc MockGitConfig) CloneOrPull(a string, b string) (map[string]struct{}, error) {
-	if mgc.cloneorpull != nil {
-		return mgc.cloneorpull(a, b)
-	}
-	return map[string]struct{}{}, nil
+	args := mgc.Called(a, b)
+	return args.Get(0).(map[string]struct{}), args.Error(1)
```

Comment:
I would propose against this change. The author advises that this is prone to panic (rightfully so) here: https://godoc.org/github.com/stretchr/testify/mock and having extra checks to see if the object is nil or not seems bit of an overkill.